kinode/packages/kns_indexer/kns_indexer/src/lib.rs

```diff
@@ -326,72 +453,27 @@ fn handle_log(our: &Address, state: &mut State, log: &eth::Log) -> anyhow::Resul
                     routers: Vec::new(),
                 },
             );
-            full_name
         }
         kimap::contract::Note::SIGNATURE_HASH => {
             let decoded = kimap::contract::Note::decode_log_data(log.data(), true).unwrap();
-
-            let note = String::from_utf8(decoded.label.to_vec())?;
+            let note: String = String::from_utf8(decoded.label.to_vec())?;
             let node_hash = decoded.parenthash.to_string();
 
             if !kimap::valid_note(&note) {
                 return Err(anyhow::anyhow!("skipping invalid note: {note}"));
             }
 
-            let Some(node_name) = get_parent_name(&state.names, &node_hash) else {
+            let Some(_node_name) = get_parent_name(&state.names, &node_hash) else {
+                pending_notes
+                    .entry(log.block_number.unwrap())
+                    .or_insert(vec![])
+                    .push(decoded);
                 return Err(anyhow::anyhow!("parent node for note not found"));
```

Comment:
We should:
1. Remove this error (maybe replace with a debug print?), or
2. Change the error text

Reply:
great point!